homebrew_challenge/public/index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="description" content="STA COVID-19 dashboard" />
     <meta name="robots" content="noindex, nofollow" />
     <link rel="icon" type="image/png" href="%PUBLIC_URL%/favicon.png" />
-    <title>Covid-19 Dashboard</title>
+    <title>Scottish COVID-19 Statistics</title>
```

Comment:
I am wondering why "Dashboard" is being dropped from the title. The website isn't really displaying statistics- it's statistics being displayed in a meaningful, digestible way. Personally, I would prefer that "Dashboard" stay in the title.

Reply:
This change just makes the html title (on the window titlebar) consistent with the display title within the webpage.

Reply:
Thanks @donal-s. This was the wrong place to start a discussion about that but it was the first time I had noticed that the name of the app wasn't "Dashboard"
I wouldn't be in favour of holding up the merge over this. If there would be a discussion about the title, then it could happen separately to this. The naming isn't related to the navbar and I would be more in favour of seeing these changes go live.